app/services/course/assessment/submission/statistics_download_service.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+require 'csv'
+class Course::Assessment::Submission::StatisticsDownloadService
+  include ApplicationFormattersHelper
+
+  class << self
+    # Downloads the submissions statistics and zip it.
+    #
+    # @param [CourseUser] course_user The course user downloading the statistics.
+    # @param [Array<Course::Assessment::Submission>] submissions The submissions to download
+    # statistics
+    # @return [String] The path to the zip file.
+    def download_and_zip(course_user, submissions)
+      service = new(course_user, submissions)
+      ActsAsTenant.without_tenant do
+        service.send(:download_to_base_dir)
+      end
+      service.send(:zip_base_dir)
+    end
+  end
+
+  private
+
+  def initialize(course_user, submissions)
+    @course_user = course_user
```

Comment:
Should this be `@current_user` instead of `@course_user`? Don't think we need any info from course_user

Reply:
Will change to current user instead.